apps/site/lib/site_web/views/trip_plan_view.ex

```diff
@@ -575,9 +575,33 @@ defmodule SiteWeb.TripPlanView do
     |> Map.put(:src, map_src)
   end
 
+  @spec readjust_itinerary_with_sl1(TripPlan.Itinerary.t()) :: TripPlan.Itinerary.t()
+  def readjust_itinerary_with_sl1(itinerary) do
```

Comment:
The preceding question might make this irrelevant, but if not, why does this readjustment need to be done in the view layer? Could this logic be incorporated such that the correct fare is determined in the first place?